internal-dns/types/src/diff.rs

```diff
@@ -155,8 +158,8 @@ impl<'a> std::fmt::Display for DnsDiff<'a> {
                     prefix,
                     match r {
                         DnsRecord::A(addr) => format!("A    {}", addr),
-                        DnsRecord::Aaaa(addr) => format!("AAAA {}", addr),
-                        DnsRecord::Srv(Srv { port, target, .. }) => {
+                        DnsRecord::AAAA(addr) => format!("AAAA {}", addr),
+                        DnsRecord::SRV(Srv { port, target, .. }) => {
```

Comment:
Not sure where exactly to attach this comment - do we like `AAAA`/`SRV` instead of `Aaaa`/`Srv`? It seems particularly in contrast in cases like this where we have `SRV(Srv { ... })`.

Reply:
Fixed. (openapi-lint complains about `Aaaa`, so I handled that with a `serde(rename)`.)